mlir/include/mlir/Analysis/Presburger/Matrix.h

```diff
@@ -239,6 +236,36 @@ class IntMatrix : public Matrix<MPInt> {
   /// Divide the columns of the specified row by their GCD.
   /// Returns the GCD of the columns of the specified row.
   MPInt normalizeRow(unsigned row);
+
+  // Return the integer inverse of the matrix, leaving the calling object
+  // unmodified.
```

Comment:
I don't know what integer inverse is; please document what is being returned exactly and when the optional is empty.